lesson_05/homework_05_2.py

```diff
@@ -0,0 +1,43 @@
+# Given list of tuples (name, surname, age, profession, City location)
+# 1 - Add your new record o the beginning of the given list
+# 2 - In modified list swap elements with indexes 1 and 5 (1<->5). Print result
+# 3 - check that all people in modified list with records indexes 6, 10, 13
+#   have age >=30. Print condition check result
+
+people_records = [
+  ('John', 'Doe', 28, 'Engineer', 'New York'),
+  ('Alice', 'Smith', 35, 'Teacher', 'Los Angeles'),
+  ('Bob', 'Johnson', 45, 'Doctor', 'Chicago'),
+  ('Emily', 'Williams', 30, 'Artist', 'San Francisco'),
+  ('Michael', 'Brown', 22, 'Student', 'Seattle'),
+  ('Sophia', 'Davis', 40, 'Lawyer', 'Boston'),
+  ('David', 'Miller', 33, 'Software Developer', 'Austin'),
+  ('Olivia', 'Wilson', 27, 'Marketing Specialist', 'Denver'),
+  ('Daniel', 'Taylor', 38, 'Architect', 'Portland'),
+  ('Grace', 'Moore', 25, 'Graphic Designer', 'Miami'),
+  ('Samuel', 'Jones', 50, 'Business Consultant', 'Atlanta'),
+  ('Emma', 'Hall', 31, 'Chef', 'Dallas'),
+  ('William', 'Clark', 29, 'Financial Analyst', 'Houston'),
+  ('Ava', 'White', 42, 'Journalist', 'San Diego'),
+  ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix')
+]
+
+#1
+people_records.insert(0,('Ron', 'Sesh', 28, 'Desiner', 'Paris'))
+
+#2
+people_records[1],people_records[5] = people_records[5],people_records[1]
+
+#3
+res_check = 0
+res_check_list = []
+for i, person in enumerate(people_records):
+    if person[2] >= 30 and i in (6, 10, 13):
```

Comment:
since indices for verification are known, you can iterate over them [6, 10, 13] and then check...